### ISSUE TITLE
Fix QCPI

### ISSUE THREAD

**Issue by SeanMatthewNolan** [Rapid-Design-of-Systems-Laboratory/beluga#49]
Changes to the bvp structure appear to have broken QCPI method

**Comment by msparapa** [comment]
Yeah it's most likely now because time is adjoined as a parameter rather than a state. I'm pretty sure thomas never properly coded in the parameter adjustments into QCPI so it shouldn't converge. I'll have time after we release v1 to sit down and learn QCPI, and modify it to handle parameters correctly.
